src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(toFileName, content);
+    }
```

Comment:
Divide program logic: The `getStatistic` method currently performs two operations: it reads the content from one file and writes it to another. According to the single responsibility principle, a method should do one thing and do it well. In this case, it would be better to separate the reading and writing operations into two different methods.